cactusbot/commands/magic/repeat.py

```diff
@@ -0,0 +1,42 @@
+"""Manage repeats."""
+
+from . import Command
+
+@Command.command()
+class Repeat(Command):
+    """Manage repeats."""
+
+    COMMAND = "repeat"
+
+    @Command.command()
+    async def add(self, period: r"[1-9]\d*", command: r"!?\w{1,32}", *args: False):
```

Comment:
We may be adding support for just text being repeated in the future. Not a problem, just an FYI